core/tests/loadnext/src/executor.rs

```diff
@@ -280,23 +324,36 @@ impl Executor {
             .get_balance(paymaster_address, None)
             .await?;
 
+        // TODO: decimals may differ from eth
```

Comment:
Better to create a task (if you believe it is to be resolved) or just state that in comment that the formatting supports only 18 decimals for now